README.md

```diff
@@ -10,13 +10,13 @@ You will need to have a valid `$GOPATH` set up, for more information about `$GOP
 Once you have a working `$GOPATH`, run:
 
 ```
-$ go get github.com/zmap/zgrab2
+$ go get github.com/packetloop/zgrab2
```

Comment:
I think we probably have a way to avoid those change `github.com/zmap/zgrab2` to `github.com/packetloop/zgrab2`
I used symbol link to avoid that change when debugging
```jwan@~/Start/atlas/go/src/github.com/zmap $ ls -l zgrab2
lrwxr-xr-x  1 jwan  staff  20 11 Mar 10:32 zgrab2@ -> ../packetloop/zgrab2```